### PR TITLE
Vilnius blog post: Call for volunteers and CFP reminder

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+:template: {{year}}/generic.html
+
+.. post:: April 3, 2019
+   :tags: vilnius-2019, volunteers, cfp, tickets
+
+Call for Volunteers and CFP Reminder
+====================================
+
+Greetings, fellow documentarians! The Vilnius conference just about two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
```
Comment (plaindocs) [inline docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst]:
```suggestion
Greetings, fellow documentarians! The Vilnius conference is just about two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
```

File: docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+**Tickets are selling fast!** We are not sold out yet, but our European community is showing great enthusiasm, and we expect to have another wave of registrations after we announce the presentations soon, so you might want to beat the rush and `get your ticket now <https://ti.to/writethedocs/write-the-docs-vilnius-2019>`_.
+
+Call for volunteers
+-------------------
+
+There is plenty to do during the conference, and volunteers are essential to the success of all our events! We are looking for folks who are willing to help out during the conference. If you volunteer for two or more shifts (at an average of three hours each), you will receive a free ticket for the conference.
```
Comment (plaindocs) [inline docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst:16]: Still 2 or more even though we're only having 1 day and 1 writing day?

File: docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+To register as a volunteer for the conference, please read and fill out the `Call for Volunteers form <https://forms.gle/bBfLsPTx4btzBS9YA>`_. You will receive additional information by late April regarding shifts, detailed duties, and other updates to the volunteer duties and schedules.
+
+Please email vilnius@writethedocs.org if you have any questions about volunteering for the conference.
+
+Call for Proposals reminder
+---------------------------
+
+We're excited to see such great proposals coming in, and we want to remind you to submit your proposals before **Midnight CET, 22nd April 2019**.
+
+The `Call for Proposals <https://www.writethedocs.org/conf/vilnius/2019/cfp/>`_ page has tips and examples to help inspire you.
```
Comment (plaindocs) [inline docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst]:
```suggestion
The `Call for Proposals <https://www.writethedocs.org/conf/vilnius/2019/cfp/>`_ page has tips and examples  to inspire you.
```

**Review by plaindocs — APPROVED** [review]
LGTM

**Review by aidiss — APPROVED** [review]
It looks great. Very detailed and fun to read.

File: docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst:16]: Yup, I think that's not a problem in this case

**Pull request merged**